user_guide_src/source/models/model.rst

```diff
@@ -295,6 +295,8 @@ the array's values are the values to save for that key:
 
 .. literalinclude:: model/015.php
 
+You can retrieve the last inserted row's primary key using the ``getInsertID()`` method.
+
```

Comment:
> @kenjis Thanks for the link. I think I did it.

Yes, you did it!

But this file change has nothing to do with this PR.
You squashed unrelated previous? commit?

Reply:
**!!!**,I do not know why this change was applied.
Do I close this PR and open a new PR?